backend/app/rest/api/rest_public.go

```diff
@@ -447,6 +460,64 @@ func (s *public) loadPictureCtrl(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// GET /search?site=siteID&query=queryText&limit=20&skip=10 - search documents
+func (s *public) searchQueryCtrl(w http.ResponseWriter, r *http.Request) {
+	siteID := r.URL.Query().Get("site")
+	query := r.URL.Query().Get("query")
+	sortBy := r.URL.Query().Get("sort")
+
+	limit := s.getNumericParam(r, "limit", 20)
```

Comment:
For consistency, I would propose to go with the way other functions like `listCtrl` get numeric params.